lib/emailnotifications_queue_handler.ts

```diff
@@ -173,6 +169,12 @@ async function handleNotification(
     return left(ProcessingError.PERMANENT);
   }
 
+  // converts the markdown body to HTML
+  // TODO: handle errors / validate the markdown
+  const bodyHtml = (await markdownToHtml.process(
```

Comment:
I guess that if the message is plain/text we should skip markdown parsing (or the intended result could be different from the sender intention).

Reply:
anyway, I'm thinking about providing an optional `text` attribute in `MessageContent` but until now all the examples we got from the SAS have all rich content, so not sure how much value could we have in that use case - we will see, it's not going to be too much effort to add a plain text option